src/lib/stitching/convection/link.ts

```diff
@@ -0,0 +1,38 @@
+import urljoin from "url-join"
```

Comment:
A service now has three parts:

* Link (how to connect to the graphql api via HTTP)
* Schema (what to merge, and how to transform it)
* Stitching (how to extend the final schema)

Every service will need the first two, not all may need the last.